lib/iiif_print.rb

```diff
@@ -173,29 +177,25 @@ def self.fields_for_allinson_flex(fields: allinson_flex_fields)
   def self.allinson_flex_fields
     return @allinson_flex_fields if defined?(@allinson_flex_fields)
 
-    # sql query method was refactored to active record
-    # original query:
-    # AllinsonFlex::ProfileProperty
-    #   .find_by_sql(
-    #     "SELECT DISTINCT allinson_flex_profile_texts.value AS label, " \
-    #     "allinson_flex_profile_properties.name AS name " \
-    #     "FROM allinson_flex_profile_properties " \
-    #     "JOIN allinson_flex_profile_texts " \
-    #     "ON allinson_flex_profile_properties.id = " \
-    #       "allinson_flex_profile_texts.profile_property_id " \
-    #     "WHERE allinson_flex_profile_texts.name = 'display_label'"
-    #   )
-    # In this ActiveRecord query, allinson_flex_profile_properties.indexing was added
     allinson_flex_relation = AllinsonFlex::ProfileProperty
                              .joins(:texts)
                              .where(allinson_flex_profile_texts: { name: 'display_label' })
                              .distinct
                              .select(:name, :value, :indexing)
     flex_fields = allinson_flex_relation.to_a
+    # filters out admin_only fields
+    flex_fields.reject! { |profile| profile.indexing.include?('admin_only') }
```

Comment:
This looks like something we could add in the above query instead of rejecting outright.